plugins/playbackProfile.ts

```diff
@@ -0,0 +1,264 @@
+import { Context } from '@nuxt/types/app';
+import { browserDetector } from './browserDetection';
+import {
+  getSupportedMP4VideoCodecs,
+  hasVp8Support
+} from '~/utils/mp4VideoFormats';
+import { getSupportedMP4AudioCodecs } from '~/utils/mp4AudioFormats';
+import { getSupportedVPXVideoCodecs } from '~/utils/vpxVideoFormats';
+import { getSupportedWebMAudioCodecs } from '~/utils/webmAudioFormats';
+import { getSupportedAudioCodecs } from '~/utils/audioFormats';
+import { canPlayNativeHls, hasMkvSupport } from '~/utils/transcodingFormats';
+import { getHlsVideoCodecs, getHlsAudioCodecs } from '~/utils/hlsFormats';
+import { getCodecProfiles } from '~/utils/codecProfiles';
+import {
+  DeviceProfile,
+  DirectPlayProfile,
+  DlnaProfileType,
+  TranscodingProfile,
+  EncodingContext,
+  SubtitleProfile,
+  SubtitleDeliveryMethod,
+  ResponseProfile
+} from '~/api';
+import { PluginInjection } from '~/types/utils';
+
+const physicalAudioChannels = browserDetector.isTv() ? 6 : 2;
```

Comment:
What about devices that aren't TVs but still have >2 audio channels?

Reply:
There's not really any API to detect the number of available audio channels, unfortunately.

Reply:
Ah ok, that makes sende.